src/components/NutritionModal/NutritionModal.js

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import Backdrop from "@mui/material/Backdrop";
+import Box from "@mui/material/Box";
+import Modal from "@mui/material/Modal";
+import Fade from "@mui/material/Fade";
+import Button from "@mui/material/Button";
+import ChartComponent from "../../components/PieChart/PieChart";
+
+const style = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 800,
+  bgcolor: "background.paper",
+  boxShadow: 24,
+  borderRadius: "10px",
+};
+
+export default function NutritionModal() {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  return (
+    <div>
+      <Button
+        onClick={handleOpen}
```

Comment:
When the 
<img width="319" alt="Screen Shot 2022-11-27 at 9 12 41 PM" src="https://user-images.githubusercontent.com/88150284/204198952-c6fb79ea-b658-4f2d-bb7d-dd3a435680ee.png">
nutritional information button is clicked there in an error in the console.

Reply:
Error fixed.